pkg/config/types.go

```diff
@@ -0,0 +1,36 @@
+// Package config implements the Kubernetes client for the resources in the remoteservice.openservicemesh.io API group
+package config
+
+import (
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/openservicemesh/osm/pkg/kubernetes"
+	"github.com/openservicemesh/osm/pkg/logger"
+)
+
+var (
+	log = logger.New("multiclusterservice-controller")
+)
+
+// informerCollection is the type used to represent the collection of informers for the remoteservice.openservicemesh.io API group
```

Comment:
```suggestion
// informerCollection is the type used to represent the collection of informers for the mulitclusterservice.openservicemesh.io API group
```

please update this in here and in all other places below